.travis.yml

```diff
@@ -17,18 +15,13 @@ script:
   - npm run build:rollup
   - npm --prefix=server test
 
-before_install:
-  - npm install -g npm@6.9.0
-  - node --version
-  - npm --version
-
 cache: npm
 
 install:
-  - npm ci
-  - npm --prefix=server ci
```

Comment:
Where does this happen with the new setup script? Or maybe we don't need it? Are we running server tests on Travis?

Reply:
This is hapening in the new setup script. Step 4, like 40. I've condensed it to one line. 

We should be running server tests on travis.